<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PDF</class>
 <widget class="QWidget" name="PDF">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>830</width>
    <height>1170</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>36</x>
     <y>244</y>
     <width>758</width>
     <height>266</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border-radius:8px;
font:NotoSansCJKsc-Regular;</string>
   </property>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>57</y>
      <width>571</width>
      <height>181</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;</string>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="styleSheet">
          <string notr="true">font: NotoSansCJKsc-Regular;
font-size: 17px;
 color: #1200ff;
</string>
         </property>
         <property name="text">
          <string>Operating system:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="styleSheet">
          <string notr="true">font: NotoSansCJKsc-Regular;
font-size: 17px;
 color: #1200ff;
</string>
         </property>
         <property name="text">
          <string>IP:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="styleSheet">
          <string notr="true">font: NotoSansCJKsc-Regular;
font-size: 17px;
 color: #1200ff;
</string>
         </property>
         <property name="text">
          <string>MAC:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="styleSheet">
          <string notr="true">font: NotoSansCJKsc-Regular;
font-size: 17px;
 color: #1200ff;
</string>
         </property>
         <property name="text">
          <string>System kernel version:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="styleSheet">
          <string notr="true">font: NotoSansCJKsc-Regular;
font-size: 17px;
 color: #1200ff;
</string>
         </property>
         <property name="text">
          <string>Software activation status:</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="m_systemName">
         <property name="styleSheet">
          <string notr="true">font: NotoSansCJKsc-Regular;
font-size: 17px;
 color: #1200ff;
</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="m_IP">
         <property name="styleSheet">
          <string notr="true">font: NotoSansCJKsc-Regular;
font-size: 17px;
 color: #1200ff;
</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="m_MAC">
         <property name="styleSheet">
          <string notr="true">font: NotoSansCJKsc-Regular;
font-size: 17px;
 color: #1200ff;
</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="m_kernel">
         <property name="styleSheet">
          <string notr="true">font: NotoSansCJKsc-Regular;
font-size: 17px;
 color: #1200ff;
</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="m_activeStatus">
         <property name="styleSheet">
          <string notr="true">font: NotoSansCJKsc-Regular;
font-size: 17px;
 color: #1200ff;
</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>751</width>
      <height>54</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>54</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../../../../resources/resources.qrc">:/images/pdf-head</pixmap>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="m_info">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>10</y>
      <width>331</width>
      <height>48</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font:NotoSansCJKsc-Regular;
font-size:17px;
background-color: transparent;</string>
    </property>
    <property name="text">
     <string>Device Information</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_3" native="true">
   <property name="geometry">
    <rect>
     <x>36</x>
     <y>520</y>
     <width>758</width>
     <height>640</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border-radius:8px;
font:NotoSansCJKsc-Regular;</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QWidget" name="widget_4" native="true">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>100</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: transparent;</string>
      </property>
      <widget class="QLabel" name="label_18">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>20</y>
         <width>751</width>
         <height>54</height>
        </rect>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>54</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="../../../../resources/resources.qrc">:/images/pdf-head</pixmap>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="m_pieChart">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>20</y>
         <width>331</width>
         <height>48</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font:NotoSansCJKsc-Regular;
font-size:17px;
background-color: transparent;</string>
       </property>
       <property name="text">
        <string>Pie chart of test results</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </widget>
    </item>
    <item row="2" column="0">
     <layout class="QHBoxLayout" name="m_pieChartLayout2">
      <property name="spacing">
       <number>0</number>
      </property>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="m_pieChartLayout">
      <property name="spacing">
       <number>0</number>
      </property>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>376</y>
     <width>830</width>
     <height>794</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(222, 237, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>830</width>
     <height>376</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string/>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../../../../resources/resources.qrc">:/images/pdf-home</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>472</y>
     <width>751</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../../../../resources/resources.qrc">:/images/pdf-clip</pixmap>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <zorder>label_16</zorder>
  <zorder>label_15</zorder>
  <zorder>widget_3</zorder>
  <zorder>widget</zorder>
  <zorder>label_17</zorder>
 </widget>
 <resources>
  <include location="../../../../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
